spec/statesman/adapters/active_record_queries_spec.rb

```diff
@@ -68,7 +75,7 @@ def self.initial_state
 
         it { is_expected.to include model }
         it { is_expected.not_to include other_model }
-        its(:to_sql) { is_expected.to include('most_recent') }
+        its(:to_sql) { is_expected.to include('most_recent =') }
```

Comment:
Good shout - looks like this was a bad spec. I'll update to make it `'.most_recent'`.